Translated/Dialog/Deirdre/ssn_014270.csv

```diff
@@ -1,6 +1,6 @@
 name_7400#0,"""Deirdre"""
 ssn_014270_0010#0,"""Hey, ain't seen you around here before. Newly arrived to Central City, are ya?"""
-ssn_014270_0020#0,"""You should come see me after you get some notches on your belt. Might be as I can teach you a thing or two."""
```

Comment:
i already told you this was intentional on my part. pls leave my fluff alone

Reply:
Whatever it is trying to convey is coming off as bad English.

Reply:
It's a perfectly fine colloquialism. Revert this.

Reply:
deridre doesn't care if you think she uses bad english. she is a tomboy queen and i will convey her message 😤